.gitmodules

```diff
@@ -16,3 +16,9 @@
 	path = esp_newlib_port
```

Comment:
This patch should probably be split into three, each doing one thing:
- add stdcpp port
- fix ld-script-irom.patch
- add sgi stl tests.